crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -841,7 +841,7 @@ impl ScheduleGraph {
 
     /// Returns `true` if the given system set is part of the graph. Otherwise, returns `false`.
     pub fn contains_set(&self, set: impl SystemSet) -> bool {
-        self.system_sets.ids.contains_key(&set.intern())
+        self.system_sets.ids.contains_key(&set as &dyn SystemSet)
```

Comment:
As long as we're converting to `dyn`, would it make sense to change the function to take a `&dyn SystemSet` instead of `impl SystemSet` and avoid some monomorphization?  Callers would need to add `&`, but the `dyn` conversion would be implicit.